src/uiStrings.ts

```diff
@@ -59,6 +60,7 @@ export class UiStrings {
     static readonly TreeitemLabelDefinitions = vscode.l10n.t("Definitions");
     static readonly TreeitemLabelVersions = vscode.l10n.t("Versions");
     static readonly TreeitemLabelEnvironments = vscode.l10n.t("Environments");
+    static readonly SelectApiEnvironment = vscode.l10n.t("Select or Create an API Environment");

```

Comment:
No 'Create'

Reply:
updated to "Select an existing API Environment"